auth/standard_role_manager.cc

```diff
@@ -26,6 +26,7 @@
 #include "db/consistency_level_type.hh"
 #include "exceptions/exceptions.hh"
 #include "log.hh"
+#include "seastar/core/future.hh"
```

Comment:
Is this header needed? There is a lot of stuff imported from seastar already, I'd be surprised if the `future.hh` header weren't included by one of them already.

If you want to make this include explicit then maybe move it higher near the other seastar imports and use angled brackets for consistency.

Reply:
some sort of auto-import I think, can remove